src/lib/config.ts

```diff
@@ -116,9 +116,22 @@ const config = convict({
     doc:
       'An ISO 4217 currency code to replace other values with for demo purposes',
     format: '*',
-    default: 'XTS',
+    default: 'TZS',
     env: 'DEMO_CURRENCY',
   },
+  simulatorDefaultAuthChannel: {
+    doc: 'The auth channel for the local simulator to use. Defaults to OTP',
```

Comment:
default doesn't match with description

Reply:
Well spotted. I'll get that in the next one.